docs/tutorial/notebook/Inference_on_a_custom_data.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Add (commented) command to download the weights directly in the notebook:</p><p> </p><p><code> # from urllib.request import urlretrieve</code></p><p><code> # urlretrieve(weights_url, filename=weights_file)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>